content/blog/minecraft-data-packs-scoreboards/index.md

```diff
@@ -0,0 +1,348 @@
+---
+{
+	title: "Minecraft Data Pack Programming: Scoreboard Usage",
+	description: "Learn data pack development in Minecraft - using player scoreboards, variables, and operations!",
+	published: '2022-08-12T23:20:03.284Z',
+	authors: ['fennifith'],
+	tags: [],
+	attached: [],
+	license: 'cc-by-nc-sa-4',
+	series: "Minecraft Data Pack Programming",
+	order: 3
+}
+---
+
+> Please note: this guide specifically covers the **Java Edition** version of Minecraft. Bedrock Edition does not use data packs, but provides customization through [add-ons](https://minecraft.fandom.com/wiki/Add-on).
+
+The data packs built in this series can be found in the [unicorn-utterances/mc-datapacks-tutorial](https://github.com/unicorn-utterances/mc-datapacks-tutorial/tree/main/3-scoreboards) repository. Feel free to use it for reference as you read through these articles!
+
+# Storing scores
+
+In many data packs, you might find a need to store information that can't be directly accessed through an entity or another command. A common way to do this is through the use of *scoreboards,* which can store a table of numbers for each entity or player. These can be used to reference player statistics, such as the number of blocks mined, or keep track of arbitrary values in your code.
+
+## Creating a scoreboard
+
+We can use the subcommands of `/scoreboard objectives` to create and modify any scoreboards that we use. Let's try making a scoreboard to track the number of animals that each player has spawned through our data pack.
+
+```shell
+scoreboard objectives add fennifith.animals_spawned dummy
+```
+
+This creates an objective named `fennifith.animals_spawned` that is connected to the `dummy` game statistic. We'll talk about what that means later on, but this effectively means the scoreboard will only be modified if you set its values yourself.
+
+## Scoreboard conventions
+
+### Namespaced objective names
+
+Players often want to have multiple data packs installed in their world at once. Since all scoreboards operate globally in the world, we need to make sure that our scoreboard names will not conflict with any scoreboards used by other data packs.
+
+To accomplish this, it is common to "namespace" your scoreboard names within your data pack by adding a certain prefix. Here, I've started my scoreboard names with `fennifith.animals` to indicate that they belong to my data pack.
+
+### Creating / removing Scoreboards
+
+Typically, all data packs will create any scoreboards they need in a `load.mcfunction` function, connected to the `#minecraft:load` function tag.
```

Comment:
It would sound a lot better and more engaging if it were addressing the reader directly "Typically, you would create any scoreboards..."